tests/test_jobs.py

```diff
@@ -239,9 +239,9 @@ def test_backfill_examples(self):
                 ignore_first_depends_on_past=True)
             job.run()
 
-    def test_backfill_conf(self):
+    def _get_dummy_dag(self, dag_id):
```

Comment:
since this is not a test file, maybe move it to the top of the class?

Reply:
Moved to top of class